Alarmierung/module.php

```diff
@@ -285,4 +272,48 @@ public function GetConfigurationForm()
 
             return json_encode($formdata);
         }
+
+        private function getAlertValue($variableID, $value)
+        {
+            if ($this->profileInverted($variableID)) {
+                return boolval(!$value);
+            }
```

Comment:
Das solltest du mit in den default-Teil ziehen, da dies ja Teil der Standardbehandlung ist

Reply:
Und möchtest du die Negation nicht aus dem boolval herausziehen? So negierst du erst, was die Variable ja sowieso schon irgendwie nach Bool konvertiert und wandelst dann nochmal um